cmd/lightclient/rpc/common.go

```diff
@@ -134,3 +134,21 @@ func SendLightClientUpdatesReqV1(ctx context.Context, period uint64, client ligh
 	//err = ssz_snappy.DecodeAndRead(bytes.NewReader(message.Data), responsePacket)
 	return ssz_snappy.DecodeLightClientUpdate(message.Data)
 }
+
+func SendConsensusClientMetadataV1(ctx context.Context, client lightrpc.SentinelClient) (*cltypes.MetadataV1, error) {
```

Comment:
please name accordingly to other methods name

Reply:
as per offline discussion, we are just removing this for now. We need to get the Wire format of the Metadata updates understood because they don't have a fork digest prefix.